webapp/first_snap/content/c/package.yaml

```diff
@@ -0,0 +1,125 @@
+    - text: The base keyword specifies a special kind of snap that provides a minimal set of libraries common to most applications. It will be mounted as the root filesystem for your application.
+    - code: |
+        base: core18
+    - text: The <code>core18</code> base is recommended.
+  parts:
+    - text: |
+        Parts define what sources are needed to assemble your app. Parts can be anything: programs, libraries, or other needed assets. In this case we have one: the DOSBox source release tarball. In other cases these can point to local directories, remote git repositories or other revision control systems.
+    - text: |
+        Before building the part, the dependencies listed as <code>build-packages</code> are installed.
+    - code: |
+        parts:
+          dosbox:
+            plugin: autotools
+            source-type: tar
+            source: http://source.dosbox.com/dosboxsvn.tgz
+            build-packages:
+              - g++
+              - make
+              - libsdl1.2-dev
+              - libpng-dev
+              - libsdl-net1.2-dev
+              - libsdl-sound1.2-dev
+              - libasound2-dev
+    - text: |
+        The <code>stage-packages</code> section specifies the list of libraries required by the DOSBox snap at runtime.
+    - code: |
+            stage-packages:
+              - libsdl-sound1.2
+              - libsdl-net1.2
+              - libxcursor1
+              - libxi6
+              - libxinerama1
+              - libxrandr2
+              - libxrender1
+              - libopenal1
+              - libsndio6.1
+              - libspeex1
+              - libvorbisfile3
+              - libwayland-client0
+              - libwayland-cursor0
+              - libwayland-egl1-mesa
+              - libxkbcommon0
+              - libglu1-mesa
+              - libasound2
+              - libasyncns0
+              - libbsd0
+              - libcaca0
+              - libdbus-1-3
+              - libflac8
+              - libgcc1
+              - libgcrypt20
+              - libgl1
+              - libglvnd0
+              - libglx0
+              - libgpg-error0
+              - liblz4-1
+              - liblzma5
+              - libncursesw5
+              - libogg0
+              - libpng16-16
+              - libpulse0
+              - libsdl1.2debian
+              - libslang2
+              - libsndfile1
+              - libstdc++6
+              - libsystemd0
+              - libtinfo5
+              - libvorbis0a
+              - libvorbisenc2
+              - libwrap0
+              - libx11-6
+              - libxau6
+              - libxcb1
+              - libxdmcp6
+              - libxext6
+              - zlib1g
+    - text: |
+            The <code>autotools</code> plugin uses the standard tools - configure and make to build the part. The command <code>snapcraft help autotools</code> gives more information on how this plugin works.
```

Comment:
This feels odd here. How about making it the first thing said after the parts section is displayed?